### ISSUE TITLE
Sample-check brings in slf4j-simple dependency

### ISSUE THREAD

**Issue by lptr** [gradle/exemplar#26]
https://github.com/gradle/exemplar/blob/master/sample-check/build.gradle.kts#L15-L17

The problem with this is that it collides with Gradle's own SLF4J implementation, producing the following output whenever an integration test is executed:

```text
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [file:/Users/lptr/Workspace/gradle/gradle/out/production/logging/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/lptr/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.16/f0cacc3d21e1027c82006a7a6f6cf2cad86d2a4f/slf4j-simple-1.7.16.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.gradle.internal.logging.slf4j.OutputEventListenerBackedLoggerContext]
```

I guess it should be up to the user of this library to decide what SLF4J bindings they want to use.

**Comment by lptr** [comment]
Looks like the dependency on SLF4J is mostly removed in `master` and should be fixed by releasing 0.5.0. I guess the libraries should be removed from `Dependencies.kt` as well, but that's not relevant to fixing this issue.

**Comment by eriwen** [comment]
This is fixed in Exemplar v0.6.0
